assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -204,3 +212,31 @@
 .googlesitekit-settings-notice-ua-cutoff-warning__notice {
 	margin: 0 14px 0 0;
 }
+
+.googlesitekit-settings-notice-adsense-top-earning-pages-widget {
+	align-items: center;
+	padding-bottom: 24px;
+	padding-top: 24px;
```

Comment:
Is this based on the grid gap by chance? If so, it should probably vary with the viewport size accordingly. Let's double check, but widget padding varies by viewport.

Reply:
This is the top & bottom padding for the `SettingsNotice` component. It does use `$grid-gap-phone`, but that doesn't seem to change by viewport.